styleguide/derek/_partials/_footer.ejs

```diff
@@ -1,5 +1,5 @@
 <!--[footernav]-->
-<div class="bg-blue-gray standard-padding-full footer">
+<div class="bg-blue-gray half-padding-full footer">
```

Comment:
That's gonna require a change on the www side as well.  Can you apply that padding to `.footer` instead and then we can remove the padding classes in www in a future sprint?